client/src/components/Feed/PostTabCard.js

```diff
@@ -162,7 +162,7 @@ const PostTabCard = ({
   useEffect(() => {
     const initialTab = tabs.find((tab) => tab.title === initialPage);
     setCurrentTab(initialTab);
-  }, [cardContents, initialPage, tabs]);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
removing the dependency would cause bugs. What is the problems are you trying to fix?

Reply:
updated dependency list to `[cardContents, initialPage]`